src/operations/executors/image_executor.ts

```diff
@@ -38,6 +38,16 @@ export let executeOp: OpExecutor = (node: Node, tensorMap: NamedTensorsMap,
           images as tfc.Tensor3D | tfc.Tensor4D, [size[0], size[1]],
           alignCorners)];
     }
+    case 'resizeNearestNeighbor': {
+      const images =
+          getParamValue('images', node, tensorMap, context) as tfc.Tensor;
+      const size = getParamValue('size', node, tensorMap, context) as number[];
```

Comment:
Sorry, I may misunderstand what you mean. `resizeNearestNeighbor` receives `size` parameter as `number[]` with 2 elements. Do we need to convert `tfc.Tensor` to `number[]` instead of passing `number[]` directly?